src/main/java/no/unit/nva/model/instancetypes/Monograph.java

```diff
@@ -0,0 +1,55 @@
+package no.unit.nva.model.instancetypes;
+
+import no.unit.nva.model.pages.MonographPages;
+import nva.commons.utils.JacocoGenerated;
+
+import java.util.Objects;
+
+public class Monograph implements PublicationInstance<MonographPages> {
+
+    private MonographPages pages;
+    protected boolean peerReviewed;
+
+    protected Monograph() {
+    }
+
+    protected Monograph(MonographPages pages, boolean peerReviewed) {
+        this.pages = pages;
+        this.peerReviewed = peerReviewed;
+    }
+
+    @Override
+    public MonographPages getPages() {
+        return pages;
+    }
+
+    @Override
+    public void setPages(MonographPages pages) {
+        this.pages = pages;
+    }
+
+    @Override
+    public boolean isPeerReviewed() {
+        return peerReviewed;
+    }
+
+    @JacocoGenerated
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Monograph)) {
+            return false;
+        }
+        Monograph monograph = (Monograph) o;
+        return peerReviewed == monograph.peerReviewed
```

Comment:
This is confusing. I think it is better to use getters in the equals that the actual fields. Some fields are private some others are protected and it is hard to keep an overview on all the fields. On the contrary,  the getters are all public and reveal how the rest of the "world" sees these objects.